tests/Alert.test.jsx

```diff
@@ -116,34 +116,17 @@ describe("Alert", () => {
     const onClose = jest.fn();
     const { getByTestId } = render(
       <Alert
+        {...{ onClose }}
         isOpen
         closeOnOutsideClick={false}
         message="Alert message"
         title="Alert title"
-        onClose={onClose}
       />
     );
     await userEvent.click(getByTestId("backdrop"));
     expect(onClose).not.toHaveBeenCalled();
   });
 
-  it("should not call onSubmit while alert is closing", async () => {
```

Comment:
The test was removed because it was a false positive test. The issue of users clicking the submit button the second time was not prevented by disabling the button after closing the alert. Users were still able to click on the button while the related API call was in progress.